spec/unit/puppet/type/registry_value_spec.rb

```diff
@@ -60,12 +61,55 @@
     it 'should validate the length of the value data'
     it 'should be case-preserving'
     it 'should be case-insensitive'
```

Comment:
We can worry about this later.. but any reason to leave these since they don't do anything either?